{{cookiecutter.project_slug}}/compose/local/django/celery/beat/start

```diff
@@ -5,4 +5,41 @@ set -o nounset
 
 
 rm -f './celerybeat.pid'
+
+postgres_ready() {
```

Comment:
Right now this function is duplicated in 3 scripts: `compose/production/django/entrypoint` & in the 2 scripts here.

Is it really required to fix Travis/Gitlab? I'm thinking that we could move this function into a common/util script & source it from the others to avoid duplication.

Reply:
Not exactly required. I first used cookiecutter-django with the intention of using Docker, but I also had zero experience with Docker. It'll be nice to have this for those not experienced and wondering why celerybeat fails if you simply do `docker-compose -f local.yml up --build`.

Although, you can see some reminiscence of me original having docker-compose -f local.yml build && docker-compose -f local.yml run --rm django python manage.py migrate && docker-compose -f local.yml up -d`...

We can move this to a different PR (and this definitely was not DRY whatsoever, so I apologize :P). To back up the point of probably needing to move this to a different PR is that connection. There are several connections due to the compose files' &django anchor. Is it possible we can just run several start files under the one Django service?

Reply:
Ideally, that would be nice if it could be part of a separate PR, yes. I think it's significant enough for that, and mix fix other issues (e.g. #1743 maybe).

Also I'm working on generating a changelog from pull requests. As it is currently, the change would not appear.